composer.json

```diff
@@ -26,16 +26,16 @@
     "nettrine/cache": "^0.3.0"
   },
   "require-dev": {
-    "contributte/console": "^0.5.0",
-    "mockery/mockery": "^1.3.3",
+    "contributte/console": "^0.9.1",
```

Comment:
Is it needed to update it?

Reply:
The originally used 0.5.x supports Symfony 4 only, so no tests are run against Symfony 5 (neither 6). Bumping the version means at least some of the CI jobs are run with Symfony 5.